rasa/train.py

```diff
@@ -92,12 +111,21 @@ def train(
         print_warning(
             "No dialogue data present. Just a Rasa NLU model will be trained."
         )
-        return _train_nlu_with_validated_data(config, nlu_data_directory, output, None)
+        return _train_nlu_with_validated_data(
+            config, nlu_data_directory, output_path, None, fixed_model_name, uncompress
+        )
 
     if nlu_data_not_present:
         print_warning("No NLU data present. Just a Rasa Core model will be trained.")
         return await _train_core_with_validated_data(
-            domain, config, story_directory, output, None, kwargs
+            domain,
```

Comment:
not necessarily scope of your pr, but if it's that many params than we might consider using keyword arguments (domain=domain), so that we not mess up the order accidentally. What do you think?